framework/validators/ExistValidator.php

```diff
@@ -73,6 +73,11 @@ class ExistValidator extends Validator
      */
     public $targetAttributeJunction = 'and';
 
+    /**
+     * @var bool whether this validator can run on salve connections
+     */
+    public $canValidateOnSlaveDb =  true;
```

Comment:
> It means validator can run on slave but it doesn't specify it would run on slave. 

Yes, that's what I mean. 

First of all, the purpose of this option is to stop using salve db when validating.  If you set this option to false, you can do it. 

And considering that in the case there is no slave db config, so you can't specify to run a validator on slave db.